Gems/LmbrCentral/Code/Source/Audio/AudioSystemComponent.cpp

```diff
@@ -285,22 +285,6 @@ namespace LmbrCentral
         }
     }
 
-    ////////////////////////////////////////////////////////////////////////
-    void AudioSystemComponent::OnCrySystemInitialized(ISystem& system, const SSystemInitParams&)
-    {
-        if (IsAudioSystemInitialized())
-        {
-            AudioSystemComponentRequestBus::Handler::BusConnect();
```

Comment:
The new version doesn't appear to ever connect to the `AudioSystemComponentRequestBus`.  And this system component, in particular, doesn't need to depend on the LevelSystem notifications.  The reason this was scheduled to connect when `OnCrySystemInitialized` had occurred is that the AudioSystem was already doing a late registration of its interface (in `CSystem::Init`).  If this system component connected and there never is an `IAudioSystem` interface (due to AudioSystem gem being disabled) then it would run into problems.

Fortunately, due to changes in #11599 the `IAudioSystem` is registered much earlier - in the constructor of the Gem's system component.  That should happen well before any system components are Activated (needs verification), which means that this `AudioSystemComponent` in LmbrCentral should be able to call `AudioSystemComponentRequestBus::Handler::BusConnect()` in its `Activate()`.  If needed, also wrap the connection in `if (IsAudioSystemInitialized())`, then we can apparently remove the connection to level system notifications here.

Reply:
It does depend on level notifications, my bad, just no longer as listener registered notifications.  So we can keep the connection to `LevelSystemLifecycleNotificationBus`.

Reply:
Great catch! Fixed!